From b8987d520e1828cf70f781353b41f74580b696af Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Thu, 27 Apr 2023 13:18:12 +0800
Subject: [PATCH 49/68] Revert "drm/msm/dpu: Disallow unallocated resources to
 be returned"

This reverts commit 8dbd54d679e3ab37be43bc1ed9f463dbf83a2259.
---
 drivers/gpu/drm/msm/disp/dpu1/dpu_rm.c | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/drivers/gpu/drm/msm/disp/dpu1/dpu_rm.c b/drivers/gpu/drm/msm/disp/dpu1/dpu_rm.c
index 9483005297438..74a13ccad34c0 100644
--- a/drivers/gpu/drm/msm/disp/dpu1/dpu_rm.c
+++ b/drivers/gpu/drm/msm/disp/dpu1/dpu_rm.c
@@ -633,11 +633,6 @@ int dpu_rm_get_assigned_resources(struct dpu_rm *rm,
 				  blks_size, enc_id);
 			break;
 		}
-		if (!hw_blks[i]) {
-			DPU_ERROR("Allocated resource %d unavailable to assign to enc %d\n",
-				  type, enc_id);
-			break;
-		}
 		blks[num_blks++] = hw_blks[i];
 	}
 
-- 
2.25.1

